src/rct1.h

```diff
@@ -713,6 +713,24 @@ enum {
 	RCT1_SCENARIO_FLAG_19 = 1 << 19,
 };
 
+enum {
+	RCT1_PARK_FLAGS_PARK_OPEN = 					(1 << 0),
+	RCT1_PARK_FLAGS_SCENARIO_COMPLETE_NAME_INPUT = 	(1 << 1),
+	RCT1_PARK_FLAGS_FORBID_LANDSCAPE_CHANGES = 		(1 << 2),
+	RCT1_PARK_FLAGS_FORBID_TREE_REMOVAL = 			(1 << 3),
+	RCT1_PARK_FLAGS_SHOW_REAL_GUEST_NAMES = 		(1 << 4),
+	RCT1_PARK_FLAGS_FORBID_HIGH_CONSTRUCTION = 		(1 << 5), 	// Below tree height
+	RCT1_PARK_FLAGS_PREF_LESS_INTENSE_RIDES = 		(1 << 6),
+	RCT1_PARK_FLAGS_FORBID_MARKETING_CAMPAIGN = 	(1 << 7),
+	RCT1_PARK_FLAGS_ANTI_CHEAT_DEPRECATED = 		(1 << 8), 	// Not used anymore, used for cheat detection
+	RCT1_PARK_FLAGS_PREF_MORE_INTENSE_RIDES = 		(1 << 9),
```

Comment:
There's a gap at bit 10 here, is it not used? Perhaps some comment would be useful, noting the default expected value